client/src/app/modules/events/event-card/event-card.component.html

```diff
@@ -19,15 +27,23 @@ <h5 class="text-left w-100 fa fa-location-arrow"> Bratislava</h5>
           </div>
         </div>
       </div>
-      <div class="card-footer d-none d-sm-block">
```

Comment:
I have no clue why it was here but without it it works as usual

Reply:
It is there to dont display (d) the buttons section, when width small - cause on phones there is no hover functionality. But show buttons, when width bigger than sm. Please add them back

Reply:
Ok I get it but still it's weird... I would expect in case of smaller device to always show the buttons and not hide them. How are these cases solved in real word examples?